src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: $transition;
```

Comment:
Add a specific transition property, duration, and easing function to the transition shorthand property for better control over the transition effect.